src/pages/docs/expression-language.mdx

```diff
@@ -0,0 +1,455 @@
+## Expression Language
+
+`Expr` language - a feature in `Golem OSS`, enables users to write programs capable of manipulating worker responses, which are WebAssembly (WASM) values.
+Currently, the expression language is designed to function seamlessly with the worker-bridge of `Golem`. More information on the worker-bridge, previously known to be the API gateway can be found here.
+
+## Purpose of Expression Language
+
```

Comment:
It could be useful to take a step back and mention the big picture, something like:

> Golem lets you invoke functions on workers directly. However, sometimes you want to put a specific API in front of those workers, such as HTTP, gRPC, or GraphQL. Yet, each of these different protocols has certain expectations around the structure of requests and responses. _Rib_ lets you adapt inputs and outputs without having to change the definitions of functions on your workers. It's the glue that lets you build any type of API atop your workers.

Reply:
Should we change the actual name`Expr` to Rib.  Are introducing Rib as a language?
Can we say we are introducing Golem Rib language?

Reply:
Yes, I would do that. 'Expr Language' is too non-specific and confusing (most programmers do not know what an expression is).